dags/test_slack_notifier.py

```diff
@@ -0,0 +1,40 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from airflow.operators.python_operator import PythonOperator
+
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Test if the Slack notifier is working",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+
+def task_fail():
+    raise Exception("Task failed")
```

Comment:
Maybe this should be "Task failed successfully" or super verbose "Task failure test successfully triggered" etc?

Reply:
This could definitely get confusing for others when triggered so I'm going with your second suggestion 🙏